sos/report/plugins/aap_hub.py

```diff
@@ -17,6 +17,7 @@ class AAPAutomationHub(Plugin, RedHatPlugin):
     plugin_name = 'aap_hub'
     profiles = ('sysmgmt', 'ansible',)
     packages = ('automation-hub',)
+    services = ('nginx', 'redis',)
```

Comment:
This means that any machine that has nginx or redis service, this plugin will be run also. Is it worth triggering based of files, like `/etc/ansible-automation-platform/  and/or `/var/log/nginx/automationhub.access.log*`

Reply:
@arif-ali you are absolutely right, i missed this one. I guess the `packages` part should be enough to auto-enable this on PAH server. 

I moved the service status capture to the bottom.

Reply:
if both the redis and the nginx plugins will be triggered, is it worth duplicating the collection of the service here?

Reply:
@arif-ali i guess this plugin won't be enabled on any systems with just `nginx` or `redis` service running (i have removed the services part).